tests/cities_indicators_test.py

```diff
@@ -1,18 +1,35 @@
+from typing import List
+
 from cities_indicators.core import get_city_indicators, get_indicators, Indicator
 from cities_indicators.city import get_city_admin
+from cities_indicators.city import City
+from cities_indicators.io import read_carto_city
 
 import pandas as pd
 import pytest
+import requests
+import pandas as pd
 
 
 def get_baseline(indicator_name):
     df = pd.read_csv("fixtures/jakarta_baseline.csv")
     return df[["geo_id", indicator_name]]
 
+def get_baseline_api(cities: List[tuple[City, str]], indicator_name: str):
```

Comment:
We might not want to make our tests dependent on a hardcoded URI to an API we're still developing. If you want to test values, I think you should just save the data as CSV in the fixtures folder or as a constant in this file.

Reply:
Agree. Updated the testing function to use the `get_baseline(indicator_name)` to get data from the CSV you put in fixtures. Kept the `get_baseline_api()` function here, in case we want to pull data from API later.